### PR TITLE
Teampage

### PR DESCRIPTION
**Pull request by YurelyCamacho**
Update team's page

### CODE REVIEW

File: pages/guidelines/fund-raiser/guide-es.md [hunk 1]
```diff
@@ -11,3 +11,211 @@
 -->
 
 <!-- # [ES] Guía para Recaudación de Fondos -->
+
```
Comment (xmnlab) [inline pages/guidelines/fund-raiser/guide-es.md]: this shouldn't be here, probably you create your branch on top of an non updated branch.
could you rebase your branch on top of upstream/main pls?

@luabida could you help @YurelyCamacho in this rebase process pls?

File: pages/team.html [hunk 2]
```diff
@@ -193,4 +258,4 @@ <h5 class="card-title">MI NOMBRE</h5>
     </div>
   </div>
 </div>
--->
+-->
```
Comment (xmnlab) [inline pages/team.html]: this red symbol means that there is no empty line at the end of the file.
it is a good practice to keep an empty line at the end of the file.
the IDEs normally have an option to do it automatically.
could you add an empty line at the end of this file please?

Comment (YurelyCamacho) [reply pages/team.html]: Ya lo hice e hice push. Qué debo hacer ahora?

Comment (xmnlab) [reply pages/team.html]: esta parte ya está ok .. ahora solo falta hacer un rebase por en cima de upstream/main

**Comment by xmnlab** [comment]
@YurelyCamacho @luabida could we have this rebased today?

**Comment by xmnlab** [comment]
LGTM! Thanks for working on that!

**Pull request merged**